blog/content/post/2022-01-03-pcie_screamer.md

```diff
@@ -0,0 +1,218 @@
+---
+title: PCIe Screamer first look
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/PCIExpress.jpg
+author: igor.bagnucki
```

Comment:
@mgabryelski1 if you will update/add new sections to blog post, please add yourself like in this example: https://github.com/3mdeb/news-and-ideas#single-or-multiple-authors